### PR TITLE
new articles for partners

### PR DESCRIPTION
**Pull request by andrewBrunette**
new articles

### CODE REVIEW

File: Ecosystem Partners/Marketplace Guides/getting-started-with-symas-open-ldap-silver.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+{{{
+  "title": "Install Symas OpenLDAP Silver",
+  "date": "12/18/2015",
```
Comment (srveit) [inline Ecosystem Partners/Marketplace Guides/getting-started-with-symas-open-ldap-silver.md]: Change to `12-18-2015`

File: Ecosystem Partners/Marketplace Guides/getting-started-with-yellowfin.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+{{{
+  "title": "Yellowfin 7.1 for CenturyLink",
+  "date": "30th December 2015",
```
Comment (srveit) [inline Ecosystem Partners/Marketplace Guides/getting-started-with-yellowfin.md]: Change to `12-30-2015`

File: Ecosystem Partners/Marketplace Guides/getting-started-with-Netwrix-Auditor.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  "title": "Getting Started with Netwrix Auditor",
+  "date": "11-11-2015",
+  "author": "Netwrix Corporation",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+![Netwrix logo](http://../../images/netwrix-logo.svg)
+
+## Partner Profile
```
Comment (jim-phillips) [inline Ecosystem Partners/Marketplace Guides/getting-started-with-Netwrix-Auditor.md]: Update headers to use `###` as the minimum

File: Ecosystem Partners/Marketplace Guides/getting-started-with-yellowfin.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+To access your application from a computer outside the CenturyLink Cloud network, perform the following tasks after you receive the email notifying you that the Blueprint completed successfully:
+  1. [Add a Public IP](../../Network/how-to-add-public-ip-to-virtual-machine.md) to your server through Control Portal
+  2. [Allow incoming traffic](../../Network/how-to-add-public-ip-to-virtual-machine.md) for desired ports by clicking on the Servers Public IP through Control Portal and configuring appropriately.
+    * The default ports to access the application are: 80, 443
+  3. Visit your Public IP address in a web browser.
+  4. Click the "Register to Receive your login details" link and follow the prompts.
+
+### Deploying the Yellowfin 7.1 Template
+
+### Howe to Deploy the Partner Image
```
Comment (jim-phillips) [inline Ecosystem Partners/Marketplace Guides/getting-started-with-yellowfin.md]: typo `Howe to`

**Pull request merged**